cmd/zc_enumerator.go

```diff
@@ -68,17 +82,27 @@ func (storedObject *storedObject) isMoreRecentThan(storedObject2 storedObject) b
 }
 
 // a constructor is used so that in case the storedObject has to change, the callers would get a compilation error
-func newStoredObject(name string, relativePath string, lmt time.Time, size int64, md5 []byte, blobType azblob.BlobType) storedObject {
+func newStoredObject(name string, relativePath string, lmt time.Time, size int64, md5 []byte, blobType azblob.BlobType, containerName string) storedObject {
 	return storedObject{
 		name:             name,
 		relativePath:     relativePath,
 		lastModifiedTime: lmt,
 		size:             size,
 		md5:              md5,
 		blobType:         blobType,
+		containerName:    containerName,
 	}
 }
 
+// used to get properties in a safe, but not so verbose manner
+func defaultStringIfPointerIsNil(wanted *string, instead string) string {
+	if wanted == nil {
+		return instead
+	}
+
+	return *wanted
+}
```

Comment:
find better place since this is not related to enumerator definitions

Reply:
Done

Reply:
Changed to IffStringNotNil and placed in iff.go